cmd/stellar-rpc/internal/config/main_test.go

```diff
@@ -85,3 +86,63 @@ func TestConfigLoadFlagsDefaultValuesOverrideExisting(t *testing.T) {
 	// Check it didn't overwrite values which were not set in the flags
 	assert.Equal(t, "localhost:8000", cfg.Endpoint)
 }
+
+func TestConfigLoadNetworkOption(t *testing.T) {
+	// Generate structs networkParameters{networkName, historyArchiveURLs, networkPassphrase} for testnet and pubnet
+	networkFlagOptions := generateNetworkParameters()
+
+	for _, networkFlagOption := range networkFlagOptions {
+		var cfg Config
+
+		// Part confirming network option writes historyArchiveURLs and networkPassphrase
+		cmd := &cobra.Command{}
+		require.NoError(t, cfg.AddFlags(cmd))
+		require.NoError(t, cmd.ParseFlags([]string{
+			"--stellar-core-binary-path", "/usr/overridden/stellar-core",
```

Comment:
It's ideal to omit anything from the test that isn't relevant to what's being tested, so you can drop this line (and again below).